scripts/test-e2e-local.js

```diff
@@ -172,28 +187,47 @@ if (argv.target === 'RNTester') {
     cd('ios');
     exec('bundle install');
 
-    // TODO: we should be able to also use HERMES_ENGINE_TARBALL_PATH
-    // if we can make RNTester step generate it already so that it gets reused
-
-    // need to discern if it's main branch or release branch
-    if (baseVersion === '1000.0.0') {
-      // main branch
-      exec(`USE_HERMES=${argv.hermes ? 1 : 0} bundle exec pod install --ansi`);
-    } else {
-      // TODO: to test this, I need to apply changes on top of a release branch
-      // a release branch
-      // copy over the .hermesversion file from react-native core into the RNTestProject
-      exec(`cp -f ${repoRoot}/sdks/.hermesversion .`);
-      exec(
-        `CI=true USE_HERMES=${
-          argv.hermes ? 1 : 0
-        } bundle exec pod install --ansi`,
-      );
-    }
+    // for this scenario, we only need to create the debug build
+    // (env variable PRODUCTION defines that podspec side)
+
+    // FIXME: need to figure out how to generate the hermes stuff
+    // (can I just do a pod install for RNTester iOS and use those files? need to find where they are stored)
+    // and get the Hermes reference here for the next phase
+    const hermesDir = 'help me';
```

Comment:
As discussed on discord, we should probably call these two scripts:
- [build_mac_framework](https://github.com/facebook/react-native/blob/main/.circleci/config.yml#L1213)
- [build_ios_framework](https://github.com/facebook/react-native/blob/main/.circleci/config.yml#L1221)
Then, we need to [prepare the apple frameworks](https://github.com/facebook/react-native/blob/main/.circleci/config.yml#L1232) and finally, we can invoke [`createHermessTarball`](https://github.com/facebook/react-native/blob/main/.circleci/config.yml#L1240).

I still believe that we should also provide a way where we tell the scripts from where it can download all the stuff, saving hours of building, though.

But let's start by having a version that works!